src/components/Navigation/KebabMenu/index.tsx

```diff
@@ -52,6 +54,11 @@ export const KebabMenu = ({ onClose }: KebabMenuProps) => {
     onClose();
   };
 
+  const handleOpenDocsClick = () => {
+    sendUserActionTrackingEvent(trackingEvents.OPEN_DOCS_CLICKED);
+    openURLInDefaultBrowser("https://docs.digma.ai");
```

Comment:
Please add this URL to https://github.com/digma-ai/digma-ui/blob/main/src/constants.ts